src/applications/gqrx/receiver.cpp

```diff
@@ -1000,11 +1004,11 @@ receiver::status receiver::start_audio_recording(const std::string filename)
     // if this fails, we don't want to go and crash now, do we
     try {
 #if GNURADIO_VERSION < 0x030900
-        wav_sink = gr::blocks::wavfile_sink::make(filename.c_str(), 2,
+        wav_sink = gr::blocks::wavfile_sink::make(filename.c_str(), d_is_stereo_type_demodulator ? 2 : 1,
```

Comment:
mey-be rename `d_is_stereo_type_demodulator` to somethink like `d_demod_channels` and set directly to 1 or 2?